src/commands/build.js

```diff
@@ -824,10 +824,14 @@ export async function getDependencyPaths(fileContent, tgtMacros = []) {
       sourcePaths.push(tgtMacroPath)
     })
   }
-  const dependencies = getList(
-    '<h4> Dependencies </h4>',
-    fileContent
-  ).filter((d) => d.endsWith('.sas'))
+
+  const dependenciesHeader = fileContent.includes('<h4> SAS Macros </h4>')
```

Comment:
Should we display some sort of deprecation warning if the header is listed as `Dependencies` instead of `SAS Macros`?